WordPress/Classes/Services/PostService.m

```diff
@@ -258,7 +258,7 @@ - (void)uploadPost:(AbstractPost *)post
         [self.managedObjectContext performBlock:^{
             Post *postInContext = (Post *)[self.managedObjectContext existingObjectWithID:postObjectID error:nil];
             if (postInContext) {
-                [self markAsFailedAndDraftIfNeededWithPost:postInContext];
+                [self markAsFailedAndDraftIfNeededWithPost:post];
```

Comment:
⚠️ What is the reason for this change? I thought it should be `postInContext` since `post` may be coming from a different thread.

Reply:
Oh sorry, I shouldn't have committed this one.

I did that when I was looking for the cause, reverted.